app/templates/main/userProfile.html

```diff
@@ -78,6 +65,45 @@ <h1>{{volunteer.firstName}} {{volunteer.lastName}}</h1>
 </div>
 <div class="accordion" id="userProfile">
 
+<!-- ################# Community & Civic Engagement Minor ################ -->
+{% if g.current_user.isStudent %}
+  <div class="accordion-item">
+    <h3 class="accordion-header" id="headingZero">
+       {% set focus = "open" if visibleAccordion == "MinorTable" else "collapsed" %}
+      <button class="accordion-button {{focus}}" type="button" data-bs-toggle="collapse" 
+      data-bs-target="#collapseZero" aria-expanded="true" aria-controls="collapseZero">Community & Civic Engagement Minor</button>
+    </h3>
+      <div id="collapseZero" class="accordion-collapse collapse {{show}} " aria-labelledby="headingZero" data-bs-parent="#userProfile">
+        <div class="accordion-body">
+          <div class="row table-responsive">
+             <p for="expressInterest">Did you know we have a Community and Civic Engagement Minor? Click 
+             <a href="https://berea.smartcatalogiq.com/en/current/catalog/majors-and-minors/interdisciplinary-minor/community-and-civic-engagement-minor/" target="_blank">here</a>
+             for more information. 
+             </p>
+              <div class="form-check form-switch"class=".form-check.form-switch">
+              <strong><label class="custom-control-label" id="interestIndicatedText" for="expressInterest">
+                {% if expressInterest is none %}
+                  Are you interested in the CCE Minor?
+                {% else %} 
+                  {{volunteer.firstName}} {{volunteer.lastName}} has indicated 
```

Comment:
Let's change this from first and last name to "You have". Because only the student sees this now.